docker-compose.yml

```diff
@@ -1,15 +1,15 @@
 
 services:
   opal-user-db:
-    container_name: database
+    container_name: opal-user-db
     image: postgres:15
     restart: always
     environment:
       - POSTGRES_DB=opal-user-db
       - POSTGRES_USER=opal-user
       - POSTGRES_PASSWORD=opal-user
     ports:
-      - "5433:5433"
+      - "5433:5432"
```

Comment:
5433 is the port on your host machine
5432 is the port inside the Docker container

This means:

The service inside the Docker container is running on port 5432 (which is the default port for PostgreSQL).
This internal port 5432 is being mapped to port 5433 on your host machine.

So, to connect to this service from your host machine or any external application, you would use port 5433.
For example:

If you're connecting to this database from your application code running on your host machine, you would connect to localhost:5433
If you're using a database management tool on your host machine, you would also connect to port 5433

I will suggest to keep these same to avoid any issues/confusion.